proto/src/prost/tendermint.abci.rs

```diff
@@ -0,0 +1 @@
quest () , path , codec) . await } pub async fn list_snapshots (& mut self , request : impl tonic :: IntoRequest < :: tendermint_proto :: abci :: RequestListSnapshots > ,) -> Result < tonic :: Response < :: tendermint_proto :: abci :: ResponseListSnapshots > , tonic :: Status > { self . inner . ready () . await . map_err (| e | { tonic :: Status :: new (tonic :: Code :: Unknown , format ! ("Service was not ready: {}" , e . into ())) }) ? ; let codec = tonic :: codec :: ProstCodec :: default () ; let path = http :: uri :: PathAndQuery :: from_static ("/tendermint.abci.ABCIApplication/ListSnapshots") ; self . inner . unary (request . into_request () , path , codec) . await } pub async fn offer_snapshot (& mut self , request : impl tonic :: IntoRequest < :: tendermint_proto :: abci :: RequestOfferSnapshot > ,) -> Result < tonic :: Response < :: tendermint_proto :: abci :: ResponseOfferSnapshot > , tonic :: Status > { self . inner . ready () . await . map_err (| e | { tonic :: Status :: new (tonic :: Code :: Unknown , format ! ("Service was not ready: {}" , e . into ())) }) ? ; let codec = tonic :: codec :: ProstCodec :: default () ; let path = http :: uri :: PathAndQuery :: from_static ("/tendermint.abci.ABCIApplication/OfferSnapshot") ; self . inner . unary (request . into_request () , path , codec) . await } pub async fn load_snapshot_chunk (& mut self , request : impl tonic :: IntoRequest < :: tendermint_proto :: abci :: RequestLoadSnapshotChunk > ,) -> Result < tonic :: Response < :: tendermint_proto :: abci :: ResponseLoadSnapshotChunk > , tonic :: Status > { self . inner . ready () . await . map_err (| e | { tonic :: Status :: new (tonic :: Code :: Unknown , format ! ("Service was not ready: {}" , e . into ())) }) ? ; let codec = tonic :: codec :: ProstCodec :: default () ; let path = http :: uri :: PathAndQuery :: from_static ("/tendermint.abci.ABCIApplication/LoadSnapshotChunk") ; self . inner . unary (request . into_request () , path , codec) . await } pub async fn apply_snapshot_chunk (& mut self , request : impl tonic :: IntoRequest < :: tendermint_proto :: abci :: RequestApplySnapshotChunk > ,) -> Result < tonic :: Response < :: tendermint_proto :: abci :: ResponseApplySnapshotChunk > , tonic :: Status > { self . inner . ready () . await . map_err (| e | { tonic :: Status :: new (tonic :: Code :: Unknown , format ! ("Service was not ready: {}" , e . into ())) }) ? ; let codec = tonic :: codec :: ProstCodec :: default () ; let path = http :: uri :: PathAndQuery :: from_static ("/tendermint.abci.ABCIApplication/ApplySnapshotChunk") ; self . inner . unary (request . into_request () , path , codec) . await } } impl < T : Clone > Clone for AbciApplicationClient < T > { fn clone (& self) -> Self { Self { inner : self . inner . clone () , } } } impl < T > std :: fmt :: Debug for AbciApplicationClient < T > { fn fmt (& self , f : & mut std :: fmt :: Formatter < '_ >) -> std :: fmt :: Result { write ! (f , "AbciApplicationClient {{ ... }}") } } }
```

Comment:
Is there a way to not generate these `proto/src/prost/tendermint*` empty files?

Reply:
It seems that the empty files related to tendermint are dependencies in other files, and that's why they get generated. I manually removed the empty ones from this commit. I left the abci.rs.